Formula/patchelf.rb

```diff
@@ -0,0 +1,25 @@
+class Patchelf < Formula
+  desc "Modify dynamic ELF executables"
+  homepage "https://nixos.org/patchelf.html"
+  url "https://nixos.org/releases/patchelf/patchelf-0.9/patchelf-0.9.tar.gz"
+  sha256 "f2aa40a6148cb3b0ca807a1bf836b081793e55ec9e5540a5356d800132be7e0a"
+
+  head do
+    url "https://github.com/NixOS/patchelf.git"
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+  end
+
+  def install
+    system "./bootstrap.sh" if build.head?
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    assert_match "syntax", shell_output("#{bin}/patchelf --help 2>&1")
```

Comment:
We need a test that exercises the functionality. For the libelf test, I wrote a minimal ELF file, perhaps you could use that?